app/models/form.rb

```diff
@@ -26,9 +24,7 @@ def draft?
   end
 
   def status
-    return STATUSES[:live] if live?
-
-    STATUSES[:draft]
+    live? ? :live : :draft
```

Comment:
Love this refactor. we can do one better in the future and actually calculate it in status in forms-api